app/models/casa_case.rb

```diff
@@ -1,3 +1,5 @@
+require_relative "../validators/court_report_validator"
```

Comment:
@Rockenfels I don't think this line is necessary since everything in the `app` directory is loaded by default -- can you try it without this, or did you already hit an error without it?

Reply:
@littleforest you're absolutely right that the `require_relative` is unneeded-- I removed it and committed the change.